app/controllers/invites_controller.rb

```diff
@@ -3,4 +3,23 @@ def show
     @invite = Invite.where(:code => params[:id]).first!
     session[:invite_id] = @invite.id
   end
+
+  def create
```

Comment:
We got a bit of code duplication with GroupsController, we'll need to refactor that later